README.md

```diff
@@ -1,21 +1,39 @@
-# 1Password Helm Charts
+<!-- Image sourced from https://blog.1password.com/introducing-secrets-automation/ -->
+<img alt="" role="img" src="https://blog.1password.com/posts/2021/secrets-automation-launch/header.svg"/>
 
-This repository hosts the official 1Password Helm Charts.
+<div align="center">
+	<h1>1Password Helm Charts</h1>
+	<p>This repository hosts the official 1Password Helm Charts.</p>
+	<a href="/QUICKSTART.md">
+		<img alt="Get started" src="https://user-images.githubusercontent.com/45081667/226940040-16d3684b-60f4-4d95-adb2-5757a8f1bc15.png" height="37"/>
+	</a>
+</div>
 
-## Installation Guide
+---
 
-### Install Helm
+## ✨ Quickstart
 
-Get the latest [Helm release](https://github.com/kubernetes/helm#install).
+1. Get the latest [Helm release](https://github.com/kubernetes/helm#install).
+2. Add the 1password repository to be able to download and install all the charts from this repository:
+   ```
+   helm repo add 1password https://1password.github.io/connect-helm-charts
+   ```
 
-### Add Repository
-The following command allows you to download and install all the charts from this repository:
+## 📦 Available Charts
 
-```
-helm repo add 1password https://1password.github.io/connect-helm-charts
-```
+- [1Password Connect and Kubernetes Operator](./charts/connect)
+- [1Password Secrets Injector](./charts/secrets-injector)
 
-## Available Charts
+## 💙 Community & Support
 
-* [1Password Connect and Kubernetes Operator](./charts/connect)
-* [1Password Secrets Injector](./charts/secrets-injector)
+- File an [issue](correct_link) for bugs and feature requests.
```

Comment:
Please put the right link here `https://github.com/1Password/connect-helm-charts/issues`